flo-workflow/src/main/java/com/spotify/flo/EvalContext.java

```diff
@@ -84,10 +84,10 @@
       // Call preRun on task contexts after all args are evaluated
       task.contexts().forEach(tc -> tc.preRun(task));
 
-      final Optional<TaskOperator> operator = task.contexts().stream()
+      final TaskOperator operator = task.contexts().stream()
```

Comment:
Why this change?

Reply:
`j.u.Optional` is not serializable

Reply:
Some interesting discussion about this: https://stackoverflow.com/a/24564612

Gist of it is that `Optional<>` was never meant to be stored as a field. I sympathize with that but seems they lost that war because _everyone_ is doing that. Surprisingly unpragmatic decision.